src/dsp_tools/commands/ingest_xmlupload/upload_xml.py

```diff
@@ -0,0 +1,67 @@
+from datetime import datetime
+
+from lxml import etree
+
+from dsp_tools.commands.ingest_xmlupload.apply_ingest import get_mapping_dict_from_file, replace_filepath_with_sipi_uuid
+from dsp_tools.commands.xmlupload.upload_config import UploadConfig
+from dsp_tools.commands.xmlupload.xmlupload import xmlupload
+from dsp_tools.models.exceptions import InputError
+from dsp_tools.utils.create_logger import get_logger
+
+logger = get_logger(__name__)
+
+
+def ingest_xmlupload(
+    xml_file: str,
+    user: str,
+    password: str,
+    dsp_url: str,
+    sipi_url: str,
+) -> None:
+    """
+    This function reads an XML file
+    and imports the data described in it onto the DSP server,
+    using the fast XML upload method.
```

Comment:
"fast XML upload" has quasi become a terminus for the workflow with the local processing, which is not what we do here. So maybe this should be re-phrased?

Reply:
My bad, I copy pasted this function and did not catch that, thanks.